src/class-parsely.php

```diff
@@ -129,6 +131,52 @@ public function __construct() {
 		add_action( 'wp_enqueue_scripts', [ $this, 'wp_parsely_style_init' ] );
 	}
 
+	/**
+	 * Include a link to the statistics page for an article in the wp-admin Posts List.
+	 * If the post object is the "front page," this will include the main dashboard link instead.
+	 *
+	 * @see https://developer.wordpress.org/reference/hooks/page_row_actions/
+	 * @see https://developer.wordpress.org/reference/hooks/post_row_actions/
+	 * @param [string[]] $actions  The list of actions to which this function will append
+	 * @param [WP_Post]  $post     The individual post object in the list
+	 * @return $actions            The filtered list of actions
+	 */
+	public function row_actions_add_parsely_link( $actions, $post ) {
+		if ( ! is_array( $actions ) ) {
+			return $actions;
+		}
+
```

Comment:
I think we should also check for the post type being `public` as well - if it's not accessible from the front-end, then it can't be in Parse.ly. A post could be non-public post type and still have a `publish` post status (WPCOM Legacy Redirector may go that route, to distinguish active from non-active redirects).

Reply:
Good suggestion.

It would be nice to just use [`is_post_publicly_viewable`](https://developer.wordpress.org/reference/functions/is_post_publicly_viewable/), but it was added in WP 5.7, so too new for our minimum.

Even [`is_post_type_viewable`](https://developer.wordpress.org/reference/functions/is_post_type_viewable/) requires WP >= 4.4.

What do you think we should check here?

Reply:
We could always either create our own function/check, or bump the minimum to WP 4.4 (or maybe later if we're going to move it anyway).

Reply:
5e4885659b9870a9379e40eb8a1f5e54f8ee6406